R/new_regimen.R

```diff
@@ -5,7 +5,7 @@
 #' @param interval dosing interval (requires n as argument)
 #' @param n number of doses (requires interval as argument)
 #' @param times vector describing dosing times. Overrides specified times using interval and n arguments
-#' @param type either "infusion", "oral" or "bolus".
+#' @param type either "infusion", "bolus", "oral", "sc", or "im".
```

Comment:
I assume sc = subcutaneous and im = intramuscular? Maybe that will be obvious to users, but it might be worth stating it in the documentation.